kubernetes/resource_kubernetes_secret_test.go

```diff
@@ -302,6 +321,20 @@ func testAccCheckKubernetesSecretExists(n string, obj *api.Secret) resource.Test
 	}
 }
 
+const testAccKubernetesSecretConfig_dotInSecretName = `
+resource "kubernetes_secret" "test" {
+	metadata {
+	  name = "dot.test"
+	}
+  
+	data {
+	  ".start" = "foo"
+	  "mid.dle" = "foo"
+	  "end." = "foo"
```

Comment:
I don't think we run the validation function for `data` here, so these three examples would be better exercised perhaps in 3 separate `kubernetes_secret`s.

Not a big deal though as we already exercise dot in the middle of the string.